anitya/lib/backends/npmjs.py

```diff
@@ -92,3 +92,28 @@ def get_versions(cls, project):
             raise AnityaPluginException('No versions found at %s' % url)
 
         return data['versions'].keys()
+
+    @classmethod
+    def check_feed(cls):
+        ''' Return a generator over the latest 40 uploads to npmjs.org
+
+        by querying an weird JSON endpoint.
+        '''
+
+        url = 'https://registry.npmjs.org/-/all/static/today.json'
```

Comment:
you can even directly connect to their database and get realtime feed: `curl -vL "https://skimdb.npmjs.com/registry/_changes?descending=true&include_docs=true&feed=continuous"`